build/buildPlatformBinaries.sh

```diff
@@ -9,29 +9,30 @@ set -e
 declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
 
 PLATFORM_TO_BUILD=$1
+DEBIAN_FLAVOR_TO_BUILD=$2
 platformsDir="$REPO_DIR/platforms"
 
 case $PLATFORM_TO_BUILD in
 	'dotnet')
-		"$platformsDir/dotnet/getDotNetSdk.sh"
+		"$platformsDir/dotnet/getDotNetSdk.sh" $DEBIAN_FLAVOR_TO_BUILD
 		;;
 	'python')
-		"$platformsDir/python/buildPython.sh"
+		"$platformsDir/python/buildPython.sh" $DEBIAN_FLAVOR_TO_BUILD
 		;;
 	'php')
-		"$platformsDir/php/buildPhp.sh"
+		"$platformsDir/php/buildPhp.sh" $DEBIAN_FLAVOR_TO_BUILD
 		;;
 	'nodejs')
-		"$platformsDir/nodejs/getNode.sh"
+		"$platformsDir/nodejs/getNode.sh" $DEBIAN_FLAVOR_TO_BUILD
```

Comment:
Is this PR for building ubuntu sdks for all platforms, If so can we change the PR name